src/Health/IPFSDirectoryHashDAO.ts

```diff
@@ -0,0 +1,33 @@
+import { inject, injectable } from 'inversify'
+import { Collection, Db } from 'mongodb'
+
+export interface UpdateAnchorAttemptInfo {
+  readonly ipfsDirectoryHash: string
+  readonly txId: string
+}
+
+type updateAnchorAttemptsInfo = (x: UpdateAnchorAttemptInfo) => Promise<void>
+
+@injectable()
+export class IPFSDirectoryHashDAO {
+  private readonly collection: Collection
+
+  constructor(@inject('DB') db: Db) {
+    this.collection = db.collection('ipfsDirectoryHashInfo')
```

Comment:
We started passing in the collection to our DAO's instead of the DB. It allows the entry point to define the collections for each DAO.

Reply:
ack